pkg/controller/command/kms/command.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package kms
+
+import (
+	"io"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/command"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/internal/cmdutil"
+	"github.com/hyperledger/aries-framework-go/pkg/internal/logutil"
+	"github.com/hyperledger/aries-framework-go/pkg/kms/legacykms"
+)
+
+var logger = log.New("aries-framework/command/kms")
+
+// Error codes
+const (
+	// CreateKeySetError is for failures while creating key set
+	CreateKeySetError = command.Code(iota + command.KMS)
+)
+
+const (
+	// command name
+	commandName = "kms"
+
+	// command methods
+	createKeySetCommandMethod = "CreateKeySet"
+)
+
+// provider contains dependencies for the kms command and is typically created by using aries.Context().
+type provider interface {
+	LegacyKMS() legacykms.KeyManager
```

Comment:
Instead of using legacykms why not use `KMS() kms.KeyManager`?

it's currently available in the framework , legacykms will go away in the future..

Reply:
we will use legacykms until you remove it